microsoftgraph/microsoft-graph-explorer-v4
# Fix: Consent type flipping

## Overview
Closes #2438

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes
Propagation of consented permissions takes time. As a result, subsequent calls to the token endpoint may not immediately return an already consented scope. This issue is however intermittent. 


## Testing Instructions

* How to test this PR
Sign in with the staging link
Consent to permissions
Notice that the consent button no longer flips
